server/src/server.js

```diff
@@ -1,6 +1,6 @@
 import express from 'express';
 import bodyParser from 'body-parser';
-import bot from './bot';
+import bot from './bot.js';
```

Comment:
don't think you need .js extension? (We should be including it in our webpack resolve extensions)